core/src/main/java/org/apache/carbondata/core/keygenerator/columnar/impl/MultiDimKeyVarLengthEquiSplitGenerator.java

```diff
@@ -143,17 +142,18 @@ private void intialize() {
 
   private int[] convertToArray(List<Integer> list) {
```

Comment:
Yes, but this is another issue, which can be fixed in another PR. What I mean here is caller of `convertToArray` can use `collection.toArray` from java.util instead.